minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -43,6 +45,8 @@
 
     private static DockerClient dockerClient = DockerClientFactory.build();
 
+    public static final Boolean MAP_AGENT_SANDBOX = Boolean.parseBoolean(System.getenv("MINIMESOS_MAP_AGENT_SANDBOX"));
```

Comment:
All other cluster configuration parameters can be set via configuration file, some of the can be changed via `up` command line parameters. Do we need to introduce another way of configuring?